libs/components/src/next/form/datepicker/stories/vl-datepicker.stories.ts

```diff
@@ -0,0 +1,76 @@
+import { Meta } from '@storybook/web-components';
+import { html } from 'lit-html';
+import { datepickerArgs, datepickerArgTypes } from './vl-datepicker.stories-arg';
+import { story } from '@domg-wc/common-storybook';
+import { VlDatepickerComponent } from '../vl-datepicker.component';
+import { registerWebComponents } from '@domg-wc/common-utilities';
+import { formatEpoch } from './vl-datepicker.stories-util';
+import datepickerDocs from './vl-datepicker.stories-doc.mdx';
+
+registerWebComponents([VlDatepickerComponent]);
+
+export default {
+    title: 'Components-next/form/datepicker',
+    tags: ['autodocs'],
+    args: datepickerArgs,
+    argTypes: datepickerArgTypes,
+    parameters: {
+        docs: {
+            page: datepickerDocs,
+            inlineStories: false,
+        },
+    },
+    decorators: [(story: () => unknown) => html` <div style="height: 400px;">${story()}</div>`],
```

Comment:
Decorator weglaten en div met inline style in story voorzien.

Reply:
Done.